attest/core/src/types/pib.rs

```diff
@@ -50,11 +51,12 @@ impl FromBase64 for PlatformInfoBlob {
     }
 }
 
-/// The hexadecimal encoding of a PIB provided by Intel has a peculiar format---it encodes an
-/// additional 4-byte prefix to the blob, which is not actually used anywhere else.
+/// The hexadecimal encoding of a PIB provided by Intel has a peculiar
+/// format---it encodes an additional 4-byte prefix to the blob, which is not
+/// actually used anywhere else.
 //
-// FIXME: move the IAS-provided PIB+4 format into it's own type, and provide conversion to/from it
-//        instead.
+// FIXME: move the IAS-provided PIB+4 format into it's own type, and provide
+// conversion to/from it        instead.
```

Comment:
formatting

Reply:
Yep, fixed :-)

Reply:
This is a different one than the one that was fixed.